TFLite/src/init.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+def init_tflite_files():
+
+    # Absolute path to schema.fbs and the path to store the generated code
+    schema_path = '/home/shobhitbehl/ml-model-characterization/TFLite/schema.fbs'
```

Comment:
Same here as well. These paths should be a command line arguments, instead of hard-coding them.